examples/calibration/plot_compare_calibration.py

```diff
@@ -150,48 +156,84 @@ def predict_proba(self, X):
 plt.show()
 
 # %%
-# :class:`~sklearn.linear_model.LogisticRegression` returns well calibrated
-# predictions as it directly optimizes log-loss. In contrast, the other methods
-# return biased probabilities, with different biases for each method:
 #
-# * :class:`~sklearn.naive_bayes.GaussianNB` tends to push
-#   probabilities to 0 or 1 (see histogram). This is mainly
-#   because the naive Bayes equation only provides correct estimate of
-#   probabilities when the assumption that features are conditionally
-#   independent holds [2]_. However, features tend to be positively correlated
-#   and is the case with this dataset, which contains 2 features
-#   generated as random linear combinations of the informative features. These
-#   correlated features are effectively being 'counted twice', resulting in
-#   pushing the predicted probabilities towards 0 and 1 [3]_.
+# :class:`~sklearn.linear_model.LogisticRegressionCV` returns reasonably well
+# calibrated predictions despite the small training set size: its reliability
+# curve is the closest to the diagonal. The reason is two-fold: this estimator
+# directly adjusts its trainable parameter to minimize negative loglikelihood
```

Comment:
A likelihood
```suggestion
# directly adjusts its trainable parameter to minimize negative log-likelihood
```
A log-likelihood only makes sense with a distribution which we don‘t mention. Also, only very few likelihood functions can serve as proper scoring rules.
I would just always use the term log loss and not mention likelihood.

Reply:
This is the first reason and the argument would be better structured with mentioning proper scoring rule here not later.

Reply:
Alright let me see how I can reorg this.

Reply:
I pushed a new commit to reorganize the analysis: I introduce strictly proper scoring rules earlier, and I only use the log-loss name.